PWGLF/Tasks/lambdakzeroanalysisMC.cxx

```diff
@@ -85,6 +94,17 @@ struct lambdakzeroQA {
       registry.fill(HIST("hDCAPosToPV"), v0.dcapostopv());
       registry.fill(HIST("hDCANegToPV"), v0.dcanegtopv());
       registry.fill(HIST("hDCAV0Dau"), v0.dcaV0daughters());
+
+      auto mcnegtrack = v0.negTrack_as<MyTracks>().mcParticle();
+      auto mcpostrack = v0.posTrack_as<MyTracks>().mcParticle();
+      auto particleMotherOfNeg = mcnegtrack.mother0_as<aod::McParticles>();
+      bool MomIsPrimary = MC::isPhysicalPrimary(particleMotherOfNeg);
```

Comment:
I'm not sure this is supported with the current version of the O2, please check that the call to primary particle is not `particleMotherOfNeg.isPhysicalPrimary()`